sdks/java/core/src/main/java/org/apache/beam/sdk/annotations/Experimental.java

```diff
@@ -76,6 +76,12 @@
     TIMERS,
 
     /** Experimental APIs related to customizing the output time for computed values. */
-    OUTPUT_TIME
+    OUTPUT_TIME,
+
+    /**
+     * <a href="https://s.apache.org/splittable-do-fn">Splittable DoFn</a>.
+     * Do not use: API is unstable and runner support is incomplete.
+     */
+    SPLITTABLE_DO_FN,
```

Comment:
I think that these experimental options are essentially invisible. If we feel strongly about "API is unstable", we could add a pipeline option that causes `ParDo.of(sdf)` to fail.